deps/secure-socket/init.lua

```diff
@@ -33,8 +33,10 @@ return function (socket, options, callback)
     thread = coroutine.running()
   end
   bioWrap(ctx, options.server, socket, callback or function (err, ssocket)
-    return assertResume(thread, ssocket, err)
-end, options.servername)
+    if coroutine.status(thread) == 'suspended' then
```

Comment:
This may deadlock if the biowrap callback is immediately called.

Reply:
secure-socket was calling the callback twice when it fails causing a double resume attempt, should now be resolved as we only execute the callback once.